src/main/java/edu/harvard/iq/dataverse/harvest/client/ClientHarvestRun.java

```diff
@@ -5,16 +5,9 @@
  */
 package edu.harvard.iq.dataverse.harvest.client;
 
+import javax.persistence.*;
```

Comment:
I'm surprised it isn't already written at http://guides.dataverse.org/en/4.20/developers/coding-style.html but we tend not to use wildcard imports like this.